TrackerMC/src/StrawDigisFromStepPointMCs_module.cc

```diff
@@ -0,0 +1,1308 @@
+//
```

Comment:
I thought that this module was removed in an earlier commit and should not be in here any more?